spec/refresher_spec.rb

```diff
@@ -5,70 +5,70 @@
   context "guess the collection" do
 
     it "Yes, it is a Object, but which class" do
-      expect([].is_a? _fill_in_object_).to be true
-      expect(Array.new.instance_of? _FILL_ME_IN_).to be true
+      expect([].is_a? Array).to be true
```

Comment:
There are a number of options that would satisfy this assertion. What are the others?